### DISCUSSION TITLE
Mod Testing

### DISCUSSION THREAD

**Discussion by NXTler** [jediminer543/JMT-MCMT#68] [General]
Well this is not at all an Issue. I'm just trying to help out the developer by testing the mods for different versions with MCMT. You can help too and write your experience with the mods in the Card comments. To test the mods I'm using the last version of the mod and the last version of forge. Core mods are not listed because you can't test them. I always get my mods from curseforge or the official website from the mod, because this is the most likely place to look after mods someone who does not have that much experience with them. If you have any questions to my test result or want to be a member for helping test mods just ask me.

Here is the link to my trello Boards:
1.15.X
https://trello.com/b/jGEImIsb
1.16.X
https://trello.com/b/xTk6aSYa

**Comment by KubiRemPL** [comment]
I am always willing to help @NXTler 
I am here for a long time.....
If U want add me on discord KubiRemPL#6064

**Comment by NXTler** [comment]
> I am always willing to help @NXTler
> I am here for a long time.....
> If U want add me on discord KubiRemPL#6064

Thank you for the quick reply you can start by just commenting to the mods you tested. I will contect you If I have the time for it.

**Comment by KubiRemPL** [comment]
Yeah from what I saw  and tested the best performance is when using Performant with disabled multi-entites option cuz it tries to thread some things in entity but not as complete as MCMT.
World edit also works fine for my use cases.
Mekanism not fully tested but it looks OK to me (for now).

**Comment by NXTler** [comment]
> Yeah from what I saw and tested the best performance is when using Performant with disabled multi-entites option cuz it tries to thread some things in entity but not as complete as MCMT.
> World edit also works fine for my use cases.
> Mekanism not fully tested but it looks OK to me (for now).

Thanks I will update trello and will roughly test them again to see if the result changes

**Comment by jediminer543** [comment]
I was trying to setup a discord to organise this kind of thing but I kinda collapsed due to mental health issues

If you're interested I can see what I can do about getting it finished

**Comment by NXTler** [comment]
> I was trying to setup a discord to organise this kind of thing but I kinda collapsed due to mental health issues
> 
> If you're interested I can see what I can do about getting it finished

This would be great if you have time for it. I would volunteer myself for managing this kind of stuff. I will probably make some forge servers on my server for long term player testing and to see how the mods perform with bigger machines to maybe improve the performens from them.
you can add me on discord if you want: NXTler#0585

**Comment by BugmanBugman** [comment]
I would love to help too

**Comment by NXTler** [comment]
> I would love to help too

If you want to help you can start by requesting mods that are not in the list. Just comment you request in the "Mod requests here" card (you can only comment their when you have an trello account). Or you can just write youre test results to the alredy tested mods or mods that dosnt appeer to work right. When you are active enought I will add you as an member to the trello board so you can create, move and delete you own cards.
